src/runtime/hexagon/hexagon_device_api.h

```diff
@@ -79,8 +78,7 @@ class HexagonDeviceAPI final : public DeviceAPI {
 
     CHECK(runtime_hexbuffs) << "runtime_hexbuffs was not created in AcquireResources";
     if (!runtime_hexbuffs->empty()) {
-      DLOG(INFO) << "runtime_hexbuffs was not empty in ReleaseResources";
-      released_runtime_buffers = runtime_hexbuffs->current_allocations();
+      LOG(INFO) << "runtime_hexbuffs was not empty in ReleaseResources";
```

Comment:
Feels like this `LOG` could move to the destructor of the `HexagonBufferManager` and the `empty` method removed from `HexagonBufferManager` given this is the only call site.

Reply:
The only downside to moving it is that we won't know which buffer manager had allocations when it was released.